src/Roundcube/ImapClient.php

```diff
@@ -440,8 +440,10 @@ protected function eof()
      */
     protected function closeSocket()
     {
-        @fclose($this->fp);
-        $this->fp = null;
+        if ($this->fp !== null) {
+            @fclose($this->fp);
+            $this->fp = null;
+        }
```

Comment:
```suggestion
        if ($this->fp === null) {
            return;
        }
        @fclose($this->fp);
        $this->fp = null;
```
Eine klein wenig schönere Lösung 🙂